proxies/live/apiproxy/targets/sync-wrap.xml

```diff
@@ -27,8 +27,8 @@
     </Request>
   </PreFlow>
   <LocalTargetConnection>
-    <APIProxy>sync-wrap-pr-21</APIProxy>
+    <APIProxy>sync-wrap-pr-22/APIProxy>
     <ProxyEndpoint>default</ProxyEndpoint>
-    <Path>/sync-wrap-pr-21/{{ SERVICE_BASE_PATH }}</Path>
+    <Path>/sync-wrap-pr-22/{{ SERVICE_BASE_PATH }}</Path>
   </LocalTargetConnection>
```

Comment:
Needs reverting back to sync-wrap released version, once pr 22 has been merged in